components/analysis-step-version-table.js

```diff
@@ -0,0 +1,78 @@
+// node_modules
+import { TableCellsIcon } from "@heroicons/react/20/solid";
+import _ from "lodash";
+import Link from "next/link";
+import PropTypes from "prop-types";
+// components
+import { DataAreaTitle, DataAreaTitleLink } from "./data-area";
+import SeparatedList from "./separated-list";
+import SortableGrid from "./sortable-grid";
+
+const analysisStepVersionColumns = [
+  {
+    id: "@id",
+    title: "Analysis Step Versions",
+  },
+  {
+    id: "software_versions",
+    title: "Software Versions",
+    display: ({ source }) => {
+      return (
+        <SeparatedList>
+          {source.software_versions.map((version) => (
+            <Link key={version["@id"]} href={version["@id"]}>
+              {version.name}
+            </Link>
+          ))}
+        </SeparatedList>
+      );
+    },
+    isSortable: false,
+  },
+];
+
+/**
+ * Display a sortable table of the given analysis steps. Optionally display a link to a report page of
```

Comment:
really for the given analysis step *versions*.

Reply:
fixed